### PR TITLE
Small docs updates

### PR DESCRIPTION
**Pull request by aesharpe**
- Fix links
- Integrate Settings File info into Running the ETL docs page
- Remove old Settings File from docs (and indexes)
- Add some comment headers to the actual deployed settings files to clarify which settings pertain to which script

### CODE REVIEW

File: docs/dev/dev_setup.rst [hunk 1]
```diff
@@ -28,25 +28,25 @@ or Linux) and are already familiar with ``git``, GitHub, and the Unix shell.
...
+After installing your package manager, make sure it's configured to use
 `strict channel priority <https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-channels.html#>`__
 with the following commands:
 
 .. code-block:: console
 
-    $ conda update conda
-    $ conda config --set channel_priority strict
+    $ mamba update mamba
+    $ mamba config --set channel_priority strict
```
Comment (zaneselvans) [inline docs/dev/dev_setup.rst:49]: Sorry, this command doesn't actually work. Has to be `conda config ...`

File: docs/dev/run_the_etl.rst [hunk 2]
```diff
@@ -24,15 +23,111 @@ There are two main scripts that are involved in the PUDL processing pipeline:
...
-    * :ref:`install-workspace` for more on how to create a PUDL data workspace.
-    * :ref:`settings_files` for info details on the contents of the settings files.
+      For an exhaustive listing of the available parameters, see the ``etl_full.yml``
+      file.
+
+There are a few notable dependencies to be wary of when fiddling with these
+settings:
+
+- EPA CEMS cannot be loaded without EIA data unless you have existing PUDL database
+  containing EIA. This is because CEMS because it relies on IDs from EIA860
```
Comment (zaneselvans) [inline docs/dev/run_the_etl.rst:113]: copy edit "because CEMS because..."

**Review by zaneselvans — CHANGES_REQUESTED** [review]
Found one `mamba` command that has to be `conda`, and a small copyedit.

**Comment by zaneselvans** [comment]
Note that the docs build is failing because there are still references to the now removed settings page.

File: docs/dev/dev_setup.rst (hunk 1 above)
Comment (aesharpe) [reply docs/dev/dev_setup.rst:49]: ooo good catch!

**Comment by aesharpe** [comment]
> Note that the docs build is failing because there are still references to the now removed settings page.

It seems like the error is due to some :download: commands in the data_source pages:

File: docs/dev/run_the_etl.rst [hunk 3]
```diff
@@ -24,15 +23,111 @@ There are two main scripts that are involved in the PUDL processing pipeline:
...
+     - A list of the years you'd like to process CEMS data for. You should
+       only use a continuous range of years. Check the :doc:`/data_sources/epacems` page
+       for the earliest available years.
+   * - ``states``
+     - A list of the state codes you'd like to process CEMS data for. You can specify
+       ``all`` if you want to process data for all states. This may take a while!
 
 .. seealso::
 
-    * :ref:`install-workspace` for more on how to create a PUDL data workspace.
```
Comment (zaneselvans) [inline docs/dev/run_the_etl.rst:34]: Did you mean to remove this link to the `install-workspace` page?

Comment (aesharpe) [reply docs/dev/run_the_etl.rst:34]: Yeah, I added the link up higher at the beginning of the Settings section in line 32

**Pull request merged**